protocols/luci-proto-wireguard/luasrc/model/cbi/admin_network/proto_wireguard.lua

```diff
@@ -81,10 +81,22 @@ preshared_key = section:taboption(
             "cryptography for post-quantum resistance.")
 )
 preshared_key.password = true
-preshared_key.datatype = "and(base64,rangelength(44, 44))"
+preshared_key.datatype = "and(base64,rangelength(44,44))"
 preshared_key.optional = true
 
 
+fwmark = section:taboption(
+  "advanced",
+  Value,
+  "fwmark",
+  translate("Firewall Mark"),
+  translate("Optional. 32-bit hexadecimal mark for outgoing encrypted " ..
+            "packets. Enter value including leading <code>0x</code>.")
+)
+fwmark.datatype = "and(hexstring,rangelength(0,8))"
```

Comment:
Wow. I rebooted and now it does not work anymore. I'll put it in`cbi/ datatypes.lua` again. At least it worked there.